baselines/dom.generated.d.ts

```diff
@@ -13009,10 +13009,14 @@ interface ElementDefinitionOptions {
     extends: string;
 }
 
+interface ElementCreationOptions {
+    is: string;
```

Comment:
https://dom.spec.whatwg.org/#dictdef-elementcreationoptions
I think `is` is not optional. 
I want to check document that you seen.

> `[NewObject] Element createElement(DOMString localName, optional ElementCreationOptions options);`

options is optional.